qa/rpc-tests/wallet_shieldcoinbase.py

```diff
@@ -185,3 +192,19 @@ def verify_locking(first, second, limit):
         sync_mempools(self.nodes[:2])
         self.nodes[1].generate(1)
         self.sync_all()
+
+        print("activate NU5")
+        current_height = self.nodes[0].getinfo()['blocks']
+        self.nodes[0].generate(1300-current_height)
+
+        print("create a unified address")
+        account = self.nodes[0].z_getnewaccount()['account']
+        ua = self.nodes[0].z_getaddressforaccount(account)['unifiedaddress']
+
+        print("shield coinbase to the unified address")
+        result = self.nodes[0].z_shieldcoinbase("*", ua)
+        assert_equal(result["remainingUTXOs"], 50)
+
+
+if __name__ == '__main__':
+    WalletShieldCoinbaseTest('sapling').main()
```

Comment:
This entire test was effectively disabled since #3518 (October 2018), so the current PR adds back this call to the test entry point here. I don't know why the test was disabled (ping @str4d); it does pass. If reviewers prefer, I can have this PR add (actually, enable) test code only for the code changes from this PR (instead of re-enabled the entire test), although I'm unsure of the best way to do that. The test takes a somewhat long time to run, so if needed I can modify the rest of the test to be faster, although perhaps that should be done as a separate PR.

Reply:
#1318 added `wallet_shieldcoinbase_sprout.py` and `wallet_shieldcoinbase_sapling.py` that execute `WalletShieldCoinbaseTest(s).main()` in this file for `s = "sprout"` or `"sapling"`. That's why this `__main__` entry point was removed.

Reply:
Yep. This file should not be edited in this way; instead we should have a separate file that calls this one, as for Sprout and Sapling.